src/handlers/motions/motionCreated/handlers/multicall/fragments.ts

```diff
@@ -0,0 +1,25 @@
+import { Result } from 'ethers/lib/utils';
+import { ContractEvent } from '~types';
+import { moveFundsBetweenPotsMulti } from './multicallHandlers';
+
+type MultiCallHandler = ({
+  args,
+  event,
+  gasEstimate,
+}: {
+  event: ContractEvent;
+  args: Result;
+  gasEstimate: string;
+}) => Promise<void>;
+// Here we list tuples with the fragments that we are aware of that use multicall, and the corresponding handler
+// we want to fire if the fragment was included in the multicall event.
```

Comment:
As suggested in the comment, this permits us to handle all the actions we expect to be wrapped in a mutlicall separately.